### PR TITLE
Add long tail relicensing page

### PR DESCRIPTION
**Pull request by kbeyls**
Also remove/adapt out-dated information from the main relicensing page.

### CODE REVIEW

File: content/docs/relicensing.md [hunk 1]
```diff
@@ -60,19 +60,22 @@ We also have worked with our legal counsel to build several more boring pieces:
 * A corporate agreement to relicense that is available for companies to sign
   and has begun to be distributed to some of the known and/or large contributors
 
-We currently plan to install the new developer policy and add the new license in
-January 2019 after the LLVM 8.0 release has branched.
+We have installed the new developer policy and added the new license on January
```
Comment (lattner) [inline content/docs/relicensing.md]: Super nit: "We have installed" -> "We installed" or "The new developer policy was installed.."

File: content/docs/relicensing_long_tail.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+the copyright on.
+
+## Individuals
+
+When reaching out to an individual, please ask them to complete [this web
+form](https://goo.gl/forms/X4HiyYRcRHOnTSvC3). Part of that form will prompt
+them with a DocuSign agreement that they can sign online to cover anything they
+personally contributed. The web form will also collect any companies or academic
+institutions that may own rights to some of their contributions so that we can
+cover them with the corporate agreement below.
```
Comment (lattner) [inline content/docs/relicensing_long_tail.md:30]: Would it make sense to allow email for people who can't use docusign?

**Review by lattner — APPROVED** [review]
Thank you so much for updating this!

File: content/docs/relicensing.md (hunk 1 above)
Comment (tlattner) [reply content/docs/relicensing.md]: I agree. I would change to "The new license was added and changes to the developer policy were completed on January 19, 2019 after the LLVM 8.0 release was branched."

File: content/docs/relicensing.md [hunk 3]
```diff
@@ -60,19 +60,22 @@ We also have worked with our legal counsel to build several more boring pieces:
...
+19th 2019 after the LLVM 8.0 release had branched.
 
-Once that is done, we will still need need to:
+We are currently collecting relicensing agreements from copyright holders of
+contributions before 19th of January 2019. While we already have well over 90%
+of code covered, we are still working through the long tail. Help is very much
+appreciated with that. Please have a look at our
```
Comment (tlattner) [inline content/docs/relicensing.md]: I would change this to say "We are seeking volunteers to help us reach individuals and corporations who have not signed the relicense agreements. For full details on how to help, please see our ...

File: content/docs/relicensing.md (hunk 1 above)
Comment (kbeyls) [reply content/docs/relicensing.md]: Thank you, I used @tlattner suggested text in the latest update of this PR.

File: content/docs/relicensing_long_tail.md (hunk 2 above)
Comment (kbeyls) [reply content/docs/relicensing_long_tail.md:30]: That's a great suggestion - I've added text now to indicate we'll also accept an email for people who can't use docusign.

File: content/docs/relicensing.md (hunk 3 above)
Comment (kbeyls) [reply content/docs/relicensing.md]: Thanks @tlattner! I used your suggested text in the latest update.

**Pull request merged**